### DISCUSSION TITLE
\njuchapter本地编译失败

### DISCUSSION THREAD

**Discussion by tiejundong** [nju-lug/NJUThesis#113] [Q&A] [answered]
tex中是：
```
\njuchapter{XXX}
\input{chapters/XXX}
```
其中XXX无论是中英文在本地都显示`You can't use 'the letter X' after \the. \njuchapter{XXX}`

使用`\chapter{XXX}`可以顺利编译，且这个问题在南大在线LaTeX上没有出现。

本地环境是win10，TeXstudio，编译器XeLaTeX

**Comment by atxy-blip** [comment, marked as answer]
我在 0.15 或者 0.16 解决了这个问题，建议检查一下你现在本地用的版本。

**Reply by tiejundong** [reply]
更新后已解决，谢dalao
